src/ckeditor/ckeditor.component.spec.ts

```diff
@@ -470,3 +470,62 @@ describe( 'CKEditorComponent', () => {
 		} );
 	} );
 } );
+
+// (#190)
+describe( 'CKEditorComponent detached', () => {
+	let fixture: ComponentFixture<CKEditorComponent>
+
+	beforeEach( () => {
+		return TestBed.configureTestingModule( {
+			declarations: [ CKEditorComponent ]
+		} ).compileComponents();
+	} );
+
+	afterEach( () => {
+		if ( fixture ) {
+			fixture.destroy();
+		}
+	} );
+
+	it( 'should set config.delayIfDetached to true by default', async () => {
+		fixture = TestBed.createComponent( CKEditorComponent );
+		const component = fixture.componentInstance;
+
+		fixture.detectChanges();
+
+		await whenEvent( 'ready', component );
+
+		expect( component.instance.config.delayIfDetached ).toBeTrue();
+	} );
+
+	it( 'should allow overriding config.delayIfDetached', async () => {
+		fixture = TestBed.createComponent( CKEditorComponent );
+		const component = fixture.componentInstance;
+		component.config = {
+			delayIfDetached: false
+		};
+
+		fixture.detectChanges();
+
+		await whenEvent( 'ready', component );
+
+		expect( component.instance.config.delayIfDetached ).toBeFalse();
+	} );
+
+	it( 'should invoke user provided config.on.instanceReady', async () => {
+		fixture = TestBed.createComponent( CKEditorComponent );
+		const spy = jasmine.createSpy();
+		const component = fixture.componentInstance;
+		component.config = {
+			on: {
+				instanceReady: spy
+			}
+		};
+
+		fixture.detectChanges();
+
+		await whenEvent( 'ready', component );
+
+		expect( spy ).toHaveBeenCalledTimes( 1 );
+	} );
```

Comment:
I mentioned it in the comment for `ckeditor.component.ts`. If you decide to pass your own `on.instanceRead` right to the component config property - like https://github.com/ckeditor/ckeditor4-angular/blob/9dda2f70e03ca17dfc2a44eae125f192b06a9b96/src/app/simple-usage/simple-usage.component.html#L13 - you will lose it.

I've made some workaround for it in vue integration: https://github.com/ckeditor/ckeditor4-vue/blob/19f73f2c4c4b282c58ce015f8eef69534897b06b/src/ckeditor.js#L101-L116

Not sure how current angular testing tool allows customization 🤔

Reply:
But this PR uses the technique you used in Vue integration 🤔 https://github.com/ckeditor/ckeditor4-angular/pull/222/files#diff-7b88c25dedf8fde14c5848fabffd503e2dc9854c5e556904008522008b9cff02R302 + https://github.com/ckeditor/ckeditor4-angular/pull/222/files#diff-7b88c25dedf8fde14c5848fabffd503e2dc9854c5e556904008522008b9cff02R336-R338

I've just checked and user-provided `config.on.instanceReady` works correctly.

Reply:
Look like I verified it on non-rebuilded samples... 🙈  My bad - you can skip this topic in that case.